src/compat.ts

```diff
@@ -106,3 +109,55 @@ export function decodeInt64ToString(buf: Buffer): string {
 
   return negative ? `-${result}` : result;
 }
+
+export interface CustomDateTime {
```

Comment:
I'd call it `EdgeDateTime`; it's not so much custom as it is a direct representation of what's in the DB.